insertion.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+
+def insertion_sort(lst):
+    """
+    Sort a list in place using insertion sort
+    """
+    for i, item in enumerate(lst[1:], start=1):
```

Comment:
This works, but I'm curious if it's slower than just doing `for i in range(len(lst)):` (not sure)